content/sensu-go/6.2/observability-pipeline/observe-schedule/checks.md

```diff
@@ -720,6 +720,15 @@ required     | false
 type         | Array
 example      | {{< code shell >}}"env_vars": ["RUBY_VERSION=2.5.0", "CHECK_HOST=my.host.internal"]{{< /code >}}
 
+<a name="scheduler-attribute"></a>
+
+|scheduler  |     |
+------------|-----
+description | Type of scheduler that schedules the check. Sensu automatically sets the `scheduler` value and overrides any user-entered values. May be `memory` for checks scheduled in-memory, `etcd` for checks scheduled with etcd leases and watchers, or `postgres` for checks scheduled with PostgreSQL using transactions and asynchronous notification (NOTIFY).
```

Comment:
> Sensu automatically sets the `scheduler` value

I'm inferring this is automatic based on settings like PostgresConfig `enable_round_robin`. If that is correct, can we explain the connection here, or in PostgresConfig docs?

Reply:
I added some explanation based on the same inference, but I couldn't find anything in the sensu-go PR to confirm. @echlebek can you double-check the explanation I added here and make sure I didn't misrepresent how Sensu derives the scheduler values?